bin/pasta_prepare_evaluation.py

```diff
@@ -26,8 +38,7 @@
 
 from tqdm import tqdm
 
-from pypasta.LinuxMaintainers import LinuxMaintainers
-from pypasta.LinuxMailCharacteristics import load_linux_mail_characteristics
+sys.path.append(os.path.join(os.path.abspath(os.pardir), 'analyses'))
```

Comment:
I wonder if there's a better way to do the imports than the current sys.path.append...-hack that we do in PaStA... I need to check that.

Reply:
Yes, I thought about it too. Unfortunately this is the cleanest I could make it. Reason: We have under analyses, mixture of R and python, due to which it is not a package/module. Imports are unclean when we have a mix.